src/screens/TopicPageScreen/index.js

```diff
@@ -477,7 +479,7 @@ const TopicPageScreen = (props) => {
         topicId={topicId}
         ref={bottomSheetFollowRef}
         onClose={() => bottomSheetFollowRef.current.close()}
-        topicName={topicDetail.name}
+        topicName={topicDetail?.name}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The use of optional chaining here is a good practice as it ensures that the code doesn't crash if `topicDetail` is null or undefined.
```diff
-        topicName={topicDetail.name}
+        topicName={topicDetail?.name}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->